README.md

```diff
@@ -23,24 +23,11 @@ You acknowledge that your submissions to this repository are made pursuant the t
   * Create a GitHub pull request from your repository, providing as many details about your changes as possible
   * After review and acceptance one of the committers will merge the pull request.
 
-When adding **new files**, please include the following Apache v2.0 license header at the top of the file, with the fields enclosed by brackets "[]" replaced with your own identifying information **(don't include the brackets!)**:
-```java
-/**
- * Copyright (C) [XXXX] [NAME OF COPYRIGHT OWNER]
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-```
+When adding **new files**, please include the Apache v2.0 license header. From the top level directory:
+
+Run `mvn license:check -Dlicense.skip=false` to check correct header formatting.
```

Comment:
Should maven instructions be included in a separate committer guide that covers not only license, but all aspects of contributing to the APEX project (active dev branch, licensing, code style). What is the common practice for the Apache projects if any?